Confirming payout is broken with new multisig

### Description

After the bot account automatically confirms a bounty payout and the repo admin confirms on the commieth.com web ui, the funds remain in the contract and do not get transferred to the recipient.

This is suspected to be caused by incompatible code in the clojurescript side [handler](https://github.com/status-im/commiteth/blob/master/src/cljs/commiteth/handlers.cljs#L344).

Caused by 
```
2017-06-06 14:40:38,063 [async-thread-macro-35] DEBUG commiteth.scheduler - run-periodic-tasks #object[org.joda.time.DateTime 0xa3fb60b 2017-06-06T17:40:38.062Z] 
2017-06-06 14:40:38,101 [async-thread-macro-35] DEBUG commiteth.scheduler - pending payout: 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d 
2017-06-06 14:40:38,103 [async-thread-macro-35] DEBUG commiteth.eth.core - {"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d"],"id":2563} 
 {:jsonrpc 2.0, :id 2563, :result {:logsBloom 0x00000000000000000000000000000000000000004000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010800000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000400000000000000000000000000000000100000000000000000000000000000000000040000000000000000, :contractAddress nil, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :logs [{:address 0xe2216ad765b2ae34620fa6261b5ba4565867a28e, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :transactionIndex 0x0, :topics [0xe1c52dc63b719ade82e8bea94cc41a0d5d28e4aaf536adb5e9cccc9ff8c1aeda], :blockNumber 0x4d708, :logIndex 0x0, :removed false, :data 0x000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae0c75093c38b9b583ab089435fe0eb14281447657b6302fe234b644ff024a9fdfb} {:address 0xe2216ad765b2ae34620fa6261b5ba4565867a28e, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :transactionIndex 0x0, :topics [0x1733cbb53659d713b79580f79f3f9ff215f78a7c7aa45890f3b89fc5cddfbf32], :blockNumber 0x4d708, :logIndex 0x1, :removed false, :data 0xc75093c38b9b583ab089435fe0eb14281447657b6302fe234b644ff024a9fdfb000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae00000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000bbb75d03915b664e3f6a3b53b8d76dd5e4a739e000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000}], :transactionIndex 0x0, :blockNumber 0x4d708, :root 0xed052ebb0c8fd87e7d14342f3573543be5a4d1a2ea7173c36989df39cb767c5b, :from 0xc8bd65c17a7d617e033b0339c44fa2a722becae0, :gasUsed 0x2b28c, :cumulativeGasUsed 0x2b28c, :to 0xe2216ad765b2ae34620fa6261b5ba4565867a28e}} 
2017-06-06 14:40:38,104 [async-thread-macro-35] INFO  commiteth.scheduler - execution receipt for issue # 233953709 :  {:logsBloom 0x00000000000000000000000000000000000000004000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010800000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000400000000000000000000000000000000100000000000000000000000000000000000040000000000000000, :contractAddress nil, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :logs [{:address 0xe2216ad765b2ae34620fa6261b5ba4565867a28e, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :transactionIndex 0x0, :topics [0xe1c52dc63b719ade82e8bea94cc41a0d5d28e4aaf536adb5e9cccc9ff8c1aeda], :blockNumber 0x4d708, :logIndex 0x0, :removed false, :data 0x000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae0c75093c38b9b583ab089435fe0eb14281447657b6302fe234b644ff024a9fdfb} {:address 0xe2216ad765b2ae34620fa6261b5ba4565867a28e, :transactionHash 0x9b7bf95b79d1ade0a30246b6b1721a1e22caa5651e8ab30ed50e25243a9b081d, :blockHash 0x05c754fcdb13886244f31323f8bb98e1f8310dc32781baf71b54bf5534fc0619, :transactionIndex 0x0, :topics [0x1733cbb53659d713b79580f79f3f9ff215f78a7c7aa45890f3b89fc5cddfbf32], :blockNumber 0x4d708, :logIndex 0x1, :removed false, :data 0xc75093c38b9b583ab089435fe0eb14281447657b6302fe234b644ff024a9fdfb000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae00000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000bbb75d03915b664e3f6a3b53b8d76dd5e4a739e000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000}], :transactionIndex 0x0, :blockNumber 0x4d708, :root 0xed052ebb0c8fd87e7d14342f3573543be5a4d1a2ea7173c36989df39cb767c5b, :from 0xc8bd65c17a7d617e033b0339c44fa2a722becae0, :gasUsed 0x2b28c, :cumulativeGasUsed 0x2b28c, :to 0xe2216ad765b2ae34620fa6261b5ba4565867a28e} 
2017-06-06 14:40:38,112 [async-thread-macro-35] ERROR commiteth.scheduler - Scheduled task failed #error {
 :cause ERROR: value too long for type character varying(128)
 :via
 [{:type java.sql.BatchUpdateException
   :message Batch entry 0 UPDATE issues
SET confirm_hash = '000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae00000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000bbb75d03915b664e3f6a3b53b8d76dd5e4a739e000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000' ,
updated = timezone('utc'::text, now())
WHERE issue_id = 233953709  was aborted: ERROR: value too long for type character varying(128)  Call getNextException to see other errors in the batch.
   :at [org.postgresql.jdbc.BatchResultHandler handleError BatchResultHandler.java 148]}
  {:type org.postgresql.util.PSQLException
   :message ERROR: value too long for type character varying(128)
   :at [org.postgresql.core.v3.QueryExecutorImpl receiveErrorResponse QueryExecutorImpl.java 2412]}]
 :trace
 [[org.postgresql.core.v3.QueryExecutorImpl receiveErrorResponse QueryExecutorImpl.java 2412]
  [org.postgresql.core.v3.QueryExecutorImpl processResults QueryExecutorImpl.java 2125]
  [org.postgresql.core.v3.QueryExecutorImpl execute QueryExecutorImpl.java 469]
  [org.postgresql.jdbc.PgStatement executeBatch PgStatement.java 791]
  [org.postgresql.jdbc.PgPreparedStatement executeBatch PgPreparedStatement.java 1547]
  [clojure.java.jdbc$execute_batch invokeStatic jdbc.clj 426]
  [clojure.java.jdbc$execute_batch invoke jdbc.clj 419]
  [clojure.java.jdbc$db_do_execute_prepared_statement$fn__34055 invoke jdbc.clj 792]
  [clojure.java.jdbc$db_transaction_STAR_ invokeStatic jdbc.clj 613]
  [clojure.java.jdbc$db_transaction_STAR_ invoke jdbc.clj 585]
  [clojure.java.jdbc$db_transaction_STAR_ invokeStatic jdbc.clj 598]
  [clojure.java.jdbc$db_transaction_STAR_ invoke jdbc.clj 585]
  [clojure.java.jdbc$db_do_execute_prepared_statement invokeStatic jdbc.clj 791]
  [clojure.java.jdbc$db_do_execute_prepared_statement invoke jdbc.clj 778]
  [clojure.java.jdbc$db_do_prepared invokeStatic jdbc.clj 814]
  [clojure.java.jdbc$db_do_prepared invoke jdbc.clj 795]
  [clojure.java.jdbc$execute_BANG_$execute_helper__34092 invoke jdbc.clj 961]
  [clojure.java.jdbc$execute_BANG_ invokeStatic jdbc.clj 963]
  [clojure.java.jdbc$execute_BANG_ invoke jdbc.clj 943]
  [clojure.java.jdbc$execute_BANG_ invokeStatic jdbc.clj 954]
  [clojure.java.jdbc$execute_BANG_ invoke jdbc.clj 943]
  [clojure.lang.AFn applyToHelper AFn.java 156]
  [clojure.lang.AFn applyTo AFn.java 144]
  [clojure.core$apply invokeStatic core.clj 650]
  [clojure.core$apply invoke core.clj 641]
  [hugsql.adapter.clojure_java_jdbc.HugsqlAdapterClojureJavaJdbc execute clojure_java_jdbc.clj 12]
  [hugsql.adapter$eval34686$fn__34687$G__34666__34692 invoke adapter.clj 3]
  [hugsql.adapter$eval34686$fn__34687$G__34665__34698 invoke adapter.clj 3]
  [clojure.lang.Var invoke Var.java 394]
  [hugsql.core$db_fn_STAR_$y__35007 doInvoke core.clj 463]
  [clojure.lang.RestFn invoke RestFn.java 445]
  [hugsql.core$db_fn_STAR_$y__35007 invoke core.clj 453]
  [commiteth.db.core$eval37827$fn__37850 invoke core.clj 38]
  [commiteth.db.bounties$update_confirm_hash invokeStatic bounties.clj 46]
  [commiteth.db.bounties$update_confirm_hash invoke bounties.clj 43]
  [commiteth.scheduler$update_confirm_hash invokeStatic scheduler.clj 109]
  [commiteth.scheduler$update_confirm_hash invoke scheduler.clj 100]
  [commiteth.scheduler$run_periodic_tasks invokeStatic scheduler.clj 192]
  [commiteth.scheduler$run_periodic_tasks invoke scheduler.clj 184]
  [chime$chime_at$fn__65609$state_machine__20275__auto____65610$fn__65612$inst_65596__65618 invoke chime.clj 72]
  [clojure.core.async$thread_call$fn__20463 invoke async.clj 434]
  [clojure.lang.AFn run AFn.java 22]
  [java.util.concurrent.ThreadPoolExecutor runWorker ThreadPoolExecutor.java 1142]
  [java.util.concurrent.ThreadPoolExecutor$Worker run ThreadPoolExecutor.java 617]
  [java.lang.Thread run Thread.java 748]]} 
Exception in thread "async-thread-macro-35" java.lang.Error: java.sql.BatchUpdateException: Batch entry 0 UPDATE issues
SET confirm_hash = '000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae00000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000bbb75d03915b664e3f6a3b53b8d76dd5e4a739e000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000' ,
updated = timezone('utc'::text, now())
WHERE issue_id = 233953709  was aborted: ERROR: value too long for type character varying(128)  Call getNextException to see other errors in the batch.
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1148)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.BatchUpdateException: Batch entry 0 UPDATE issues
SET confirm_hash = '000000000000000000000000c8bd65c17a7d617e033b0339c44fa2a722becae00000000000000000000000000000000000000000000000000de0b6b3a7640000000000000000000000000000bbb75d03915b664e3f6a3b53b8d76dd5e4a739e000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000000' ,
updated = timezone('utc'::text, now())
WHERE issue_id = 233953709  was aborted: ERROR: value too long for type character varying(128)  Call getNextException to see other errors in the batch.
	at org.postgresql.jdbc.BatchResultHandler.handleError(BatchResultHandler.java:148)
	at org.postgresql.core.ResultHandlerDelegate.handleError(ResultHandlerDelegate.java:50)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2126)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:469)
	at org.postgresql.jdbc.PgStatement.executeBatch(PgStatement.java:791)
	at org.postgresql.jdbc.PgPreparedStatement.executeBatch(PgPreparedStatement.java:1547)
	at clojure.java.jdbc$execute_batch.invokeStatic(jdbc.clj:426)
	at clojure.java.jdbc$execute_batch.invoke(jdbc.clj:419)
	at clojure.java.jdbc$db_do_execute_prepared_statement$fn__34055.invoke(jdbc.clj:792)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:613)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:598)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:585)
	at clojure.java.jdbc$db_do_execute_prepared_statement.invokeStatic(jdbc.clj:791)
	at clojure.java.jdbc$db_do_execute_prepared_statement.invoke(jdbc.clj:778)
	at clojure.java.jdbc$db_do_prepared.invokeStatic(jdbc.clj:814)
	at clojure.java.jdbc$db_do_prepared.invoke(jdbc.clj:795)
	at clojure.java.jdbc$execute_BANG_$execute_helper__34092.invoke(jdbc.clj:961)
	at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:963)
	at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:943)
	at clojure.java.jdbc$execute_BANG_.invokeStatic(jdbc.clj:954)
	at clojure.java.jdbc$execute_BANG_.invoke(jdbc.clj:943)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.core$apply.invokeStatic(core.clj:650)
	at clojure.core$apply.invoke(core.clj:641)
	at hugsql.adapter.clojure_java_jdbc.HugsqlAdapterClojureJavaJdbc.execute(clojure_java_jdbc.clj:12)
	at hugsql.adapter$eval34686$fn__34687$G__34666__34692.invoke(adapter.clj:3)
	at hugsql.adapter$eval34686$fn__34687$G__34665__34698.invoke(adapter.clj:3)
	at clojure.lang.Var.invoke(Var.java:394)
	at hugsql.core$db_fn_STAR_$y__35007.doInvoke(core.clj:463)
	at clojure.lang.RestFn.invoke(RestFn.java:445)
	at hugsql.core$db_fn_STAR_$y__35007.invoke(core.clj:453)
	at commiteth.db.core$eval37827$fn__37850.invoke(core.clj:38)
	at commiteth.db.bounties$update_confirm_hash.invokeStatic(bounties.clj:46)
	at commiteth.db.bounties$update_confirm_hash.invoke(bounties.clj:43)
	at commiteth.scheduler$update_confirm_hash.invokeStatic(scheduler.clj:109)
	at commiteth.scheduler$update_confirm_hash.invoke(scheduler.clj:100)
	at commiteth.scheduler$run_periodic_tasks.invokeStatic(scheduler.clj:192)
	at commiteth.scheduler$run_periodic_tasks.invoke(scheduler.clj:184)
	at chime$chime_at$fn__65609$state_machine__20275__auto____65610$fn__65612$inst_65596__65618.invoke(chime.clj:72)
	at clojure.core.async$thread_call$fn__20463.invoke(async.clj:434)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	... 2 more
Caused by: org.postgresql.util.PSQLException: ERROR: value too long for type character varying(128)
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2412)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2125)
	... 43 more
```
https://github.com/status-im/commiteth/blob/master/src/clj/commiteth/eth/multisig_wallet.clj#L30 

Change needed is to extract from :data the confirmhash, that in topic 0x1733cbb53659d713b79580f79f3f9ff215f78a7c7aa45890f3b89fc5cddfbf32 is first 32 bytes

--- a/src/clj/commiteth/eth/core.clj
+++ b/src/clj/commiteth/eth/core.clj
@@ -74,7 +74,6 @@
                     (when-not (contains? params :gas)
                       {:gas
                        (estimate-gas from to value params)}))]
-    (log/debug "gas:" gas)
     (log/debug "args:" args)
     (eth-rpc
      "personal_sendTransaction"

--- a/src/clj/commiteth/eth/multisig_wallet.clj
+++ b/src/clj/commiteth/eth/multisig_wallet.clj
@@ -2,7 +2,7 @@
   (:require [commiteth.eth.core :as eth]
             [clojure.tools.logging :as log]))
 
-(def confirmation-topic "0xe1c52dc63b719ade82e8bea94cc41a0d5d28e4aaf536adb5e9cccc9ff8c1aeda")
+(def confirmation-topic "0x1733cbb53659d713b79580f79f3f9ff215f78a7c7aa45890f3b89fc5cddfbf32")
 
 (defn get-owner
   [contract index]
@@ -29,7 +29,7 @@
         confirmation-event     (first (filter has-confirmation-event logs))
         confirmation-data      (:data confirmation-event)]
     (when confirmation-data
-      (subs confirmation-data 66))))
+      (subs confirmation-data 2 66))))
 
   (defn erc20-transfer 
     [contract token to amount]
@@ -42,3 +42,4 @@
                 "0x60"
                 "0x44"
                 (eth/format-call-params "0xa9059cbb" to amount)))
+                

--- a/src/cljs/commiteth/handlers.cljs
+++ b/src/cljs/commiteth/handlers.cljs
@@ -349,7 +349,7 @@
    (let [send-transaction-fn (aget js/web3 "eth" "sendTransaction")
          payload {:from  owner-address
                   :to    contract-address
-                  :value 1
+                  :value 0
                   :data  (str "0x797af627" confirm-hash)}]
      (try
        (apply send-transaction-fn [(clj->js payload)

